File: TransCoder_tokenized_test_set_functions/python/FIND_THE_TWO_REPEATING_ELEMENTS_IN_A_GIVEN_ARRAY.py
def printRepeating ( arr, size ) :
    print ( "Repeating elements are ", end = '' )
    for i in range ( 0, size ) :
        for j in range ( i + 1, size ) :
            if arr [ i ] == arr [ j ] :
                print ( arr [ i ], end = ' ' )
    

File: TransCoder_tokenized_test_set_functions/cpp/FIND_THE_TWO_REPEATING_ELEMENTS_IN_A_GIVEN_ARRAY.cpp
void printRepeating ( int arr [ ], int size ) {
  int i, j;
  printf ( " Repeating elements are " );
  for ( i = 0;
  i < size;
  i ++ ) for ( j = i + 1;
  j < size;
  j ++ ) if ( arr [ i ] == arr [ j ] ) cout << arr [ i ] << " ";
}